libs/labelbox/tests/integration/test_labeling_frontend.py

```diff
@@ -8,21 +8,15 @@ def test_get_labeling_frontends(client):
     assert len(filtered_frontends)
 
 
-@pytest.mark.skip(
-    reason="broken due to get_projects HF for sunset-custom-editor")
 def test_labeling_frontend_connecting_to_project(project):
     client = project.client
     default_labeling_frontend = next(
         client.get_labeling_frontends(where=LabelingFrontend.name == "Editor"))
 
-    assert project.labeling_frontend(
-    ) == default_labeling_frontend  # we now have a default labeling frontend
+    assert project.labeling_frontend() is None
```

Comment:
We have reverted the back-end change that created a labeling editor automatically